### PR TITLE
[No ticket] add mocks and getMessage

### PR DESCRIPTION
**Pull request by jdcanas**
**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

**Review by kyuksel — COMMENTED** [review]
Should these be under `src/test`?

Are they intended to be used by `workbench-libs` tests or clients' tests?

**Comment by jdcanas** [comment]
they should be under src/test, maybe i'm reading it wrong? 

they are intended for client tests. We typically put mocks for wb-libs components in this repo

**Review by Qi77Qi — APPROVED** [review]

File: google2/CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Changed:
 Added:
 - Add `detachDisk`
 
-SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-google2" % "0.11-a9d31ea"`
```
Comment (Qi77Qi) [inline google2/CHANGELOG.md:16]: would be good to add some description in changelog

**Review by rtitle — APPROVED** [review]

**Comment by kyuksel** [comment]
Ah...I looked at the file path for the `kubernetesModels.scala` change and thought the others were in there too  🤦

**Review by kyuksel — APPROVED** [review]

**Pull request merged**